region/maridia/inner-yellow/Northwest Maridia Bug Room.json

```diff
@@ -516,6 +525,9 @@
       "exitCondition": {
         "leaveWithSpark": {}
       },
+      "unlocksDoors": [
+        {"types": ["ammo"], "requires": []}
```

Comment:
might need some way to get back through the morph tunnel for this strat and the next